voice/handle-user-input-with-dtmf.go

```diff
@@ -0,0 +1,61 @@
+package main
+
+import (
+	"encoding/json"
+	"io/ioutil"
+	"net/http"
+
+	"github.com/vonage/vonage-go-sdk/ncco"
+)
+
+type Dtmf struct {
+	Digits    string
+	Timed_out bool
+}
+
+type Response struct {
+	Speech            []string
+	Dtmf              Dtmf
+	From              string
+	To                string
+	Uuid              string
+	Conversation_uuid string
+	Timestamp         string
+}
+
+func answer(w http.ResponseWriter, req *http.Request) {
+	MyNcco := ncco.Ncco{}
+
+	talk := ncco.TalkAction{Text: "Hello please press any key to continue."}
+	MyNcco.AddAction(talk)
+
+	inputAction := ncco.InputAction{EventUrl: []string{"https://demo.ngrok.io/webhooks/dtmf"}, Dtmf: &ncco.DtmfInput{MaxDigits: 1}}
```

Comment:
This will make it so the user doesn't have to change the URL to make the code snippet work. It reads the host from the headers and constructs the DTMF webhook URL. 

```suggestion
	inputAction := ncco.InputAction{EventUrl: []string{"https://" + req.Host + "/webhooks/dtmf"}, Dtmf: &ncco.DtmfInput{MaxDigits: 1}}
```